package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nitro-storybook",
-  "version": "1.6.1",
+  "version": "1.7.0",
```

Comment:
@drborges you will want to change the version [here](https://github.com/powerhome/nitro-storybook/blob/master/lib/nitro_sg/version.rb) as well

Reply:
Thanks for pointing that out 👍